coalaip_bigchaindb/plugin.py

```diff
@@ -1 +1,114 @@
-# -*- coding: utf-8 -*-
+from bigchaindb_driver import BigchainDB
+from bigchaindb_driver.crypto import generate_keypair
+
+
+# FIXME: maybe this should inherit from a base plugin class provided by pycoalaip?
```

Comment:
Was toying around with the idea of having all plugins derive from a base class defining the API contract in `pycoalaip`, but not sure how useful this is in Python.